client-js/app/scripts/app.ts

```diff
@@ -351,34 +334,30 @@ window.addEventListener('WebComponentsReady', () => {
       this.plot.addData(data);
     }
 
-    stop(): Promise<void> {
+    async stop(): Promise<void> {
```

Comment:
Is it correct to just think of `async` as "Make this function return a resolved `Promise` with value equal to whatever this function returns"? Sort of like something I'd normally expect to be a decorator, but here it's a keyword instead.